src/apps/concepts/api.ts

```diff
@@ -59,6 +61,7 @@ const api = {
                 [sortDirection]: sortBy,
                 ...optionallyIncludeList("datatype", dataTypeFilters),
                 ...optionallyIncludeList("conceptClass", classFilters),
+                ...optionallyIncludeList("conceptgeneral", generalFilters),
                 ...optionallyIncludeList("source", sourceFilters),
                 timestamp: new Date().getTime(),
                 includeRetired
```

Comment:
Instead, what we want is something like `includeRetired: "Include Retired" in generalFilters`

Reply:
@ibacher thanks for the review! let me fix the proposed changes immediately.